src/OrchardCore.Modules/OrchardCore.Workflows/Views/Items/CorrelateTask.Fields.Edit.cshtml

```diff
@@ -40,6 +51,19 @@
             window.addEventListener("submit", function () {
                 textArea.value = editor.getValue();
             });
+
+            $('#@Html.IdFor(x => x.IsLiquidSyntax)').on('change', function() {
```

Comment:
I'm not sure if related to this, but the textbox is tiny for me:

![image](https://github.com/OrchardCMS/OrchardCore/assets/1976647/e496cad1-d43b-40f0-ad45-7164cbd6cb62)

Reply:
> I'm not sure if related to this, but the textbox is tiny for me:

It was implemented 2 years ago. I remember that when I tested it, the editor was bigger. You can see it on demo video attached.

Anyway, I will try to find some time to fix spotted issues on upcoming weekend. Thx @Piedone

Reply:
Thank you!